05_syntax_lab/01.py

```diff
@@ -1,5 +1,7 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print "Enter 10 numbers"
+max = int(raw_input())
+for i in range (1,10):
+    number = int(raw_input())
+    if number > max: 
+        high = number
+print "The biggeset number is: %d" % max
```

Comment:
Quite late, but welcome to the exercises!

* Could you solve this without calling ```raw_input()``` twice?
* Instead of ```%``` use the ```.format()``` for formatting a string.
* Also please rename the variable ```max``` to something that is not a keyword in Python.